### ISSUE TITLE
Backward compatibility broken for Xml Comments remarks in Swashbuckle 5.0

### ISSUE THREAD

**Issue by christopheblin** [domaindrivendev/Swashbuckle#190]
Let's do the following controller :
```c#
    [RoutePrefix("example")]
    public class TestController : ApiController
    {
        /// <summary>
        /// This comment will go into swagger to explain what the method does.
        /// </summary>
        ///<remarks>
        /// this comment will end up in implementation notes BUT ALSO in the title
        ///</remarks>
        [Route("authenticated", Name = "exampleAuthenticatedRoute")]
        [HttpGet]
        [ResponseType(typeof(string))]
        public HttpResponseMessage GetExample()
        {
             return Request.CreateResponse(HttpStatusCode.OK, "dummy");                
        }
   }
```

In 4.1.0, the implementation note appeared inside the block for the method
In 5.0.0, it appeared inside the block and on the side of the controller name.
Note that I think it was working with 5.0.0-alpha

**Comment by domaindrivendev** [comment]
So, technically this looks like a bug in the embedded version of swagger-ui (v2.1.4) as opposed to Swashbuckle itself.

I've created a corresponding ticket there ...

https://github.com/swagger-api/swagger-ui/issues/937

**Comment by christopheblin** [comment]
seems fine to me.

you've done a nice bug report on swagger-ui, I hope this will be fixed quickly :)

**Comment by domaindrivendev** [comment]
Should be fixed in latest release (5.0.1)
